doc/man1/openssl-s_server.pod.in

```diff
@@ -402,22 +390,18 @@ web browser. Cannot be used in conjunction with B<-early_data>.
 
 Emulates a simple web server. Pages will be resolved relative to the
 current directory, for example if the URL https://myhost/page.html is
-requested the file F<./page.html> will be loaded. Cannot be used in conjunction
+requested the file F<./page.html> will be loaded.
```

Comment:
You seem to merge `-WWW` and `-HTTP` documentation here which I do not object to but the text looks mixed up, you do not mention `-HTTP` and there are some duplications.

Reply:
this must have been a rebase gone wrong.  HOWEVER, I took the time to figure out what the difference was between -www -WWW and -HTTP and updated the docs to be more correct.
Additional commit pushed.